rcp-stripe-elements.php

```diff
@@ -95,4 +95,51 @@ function rcp_elements_is_update_card_page() {
 	}
 
 	return false;
-}
\ No newline at end of file
+}
+
+/**
+ * Checks if the membership can be cancelled
+ *
+ * @param bool           $can_cancel    Whether or not the membership can be cancelled.
+ * @param int            $membership_id ID of the membership being checked.
+ * @param RCP_Membership $membership    Membership object.
+ *
+ * @return bool Whether or not the membership can be cancelled.
+ */
+function rcp_elements_can_cancel( $can_cancel, $membership_id, $membership ) {
+	if ( 'stripe_elements' != $membership->get_gateway() ) {
```

Comment:
Would like to see this as a strict equality check (`!==`), and also a non-yoda condition